.bash_aliases

```diff
@@ -0,0 +1,4 @@
+# for Proficio
```

Comment:
Readme should instruct user to cat .aliases >> ~/.bash_aliases && source it, instead of potentially overwriting their existing one.